canvasapi/requester.py

```diff
@@ -111,6 +111,11 @@ def request(
             raise Conflict(response.text)
         elif response.status_code == 500:
             raise CanvasException("API encountered an error processing your request")
+        elif response.status_code > 400:
```

Comment:
Is it redundant to check > 400 _and_ 500 status codes?

Reply:
I had assumed we wanted a separate error message specifically for 500 since we had already specified one, but am open to merging the 500 and >400 responses

Reply:
I can't think of a reason to isolate 500 errors specifically, so I'm also okay with merging them together.